src/plugin.js

```diff
@@ -74,7 +74,7 @@ export function matchImageSnapshotPlugin({ path: screenshotPath }) {
   );
   const screenshotDir = screenshotPath.replace(screenshotFileName, '');
   const relativePath = screenshotDir.match(
-    new RegExp(`${screenshotsFolder}(.*)`)
+    new RegExp(`${screenshotsFolder.replace(/\\/g, '\\\\')}(.*)`)
```

Comment:
Why should we replace each single `\` with four `\`?

Reply:
It replaces each single '\' with two backslashes. On Linux you're not likely to have backslashes in your directory path but users on Windows do. However, single backslashes, which are part of paths on Windows, have special meaning in RegExp and get transformed into something different in the regular expression. So all occurences of '\' in a string must be transformed into '\\' if you want to create a RegExp from the string and keep those backslashes in path intact. 
This is no problem on Linux or with Unix style paths and at the same time resolves the issue of cypress-image-snapshot crashing on Windows because the RegExp converted 'screenshotsFolder' into something that doesn't match 'screenshotsDir' at all

Reply:
Ah that makes total sense, I think this would definitely fix the `RegExp` issue.